packages/ragbits-core/src/ragbits/core/llms/clients/local.py

```diff
@@ -74,10 +72,9 @@ async def call(
         conversation: ChatFormat,
         options: LocalLLMOptions,
         json_mode: bool = False,
-        output_schema: Optional[Type[BaseModel] | Dict] = None,
+        output_schema: type[BaseModel] | dict | None = None,
     ) -> str:
-        """
-        Makes a call to the local LLM with the provided prompt and options.
+        """Makes a call to the local LLM with the provided prompt and options.
```

Comment:
Why are these docstrings formatted this way? In some places they are formatted with a new line like this:
```
"""
some docstring
"""
```
I think we should stick to one format (preferably the latter).